core/src/test/scala/com/pingcap/tispark/overflow/DecimalOverflowSuite.scala

```diff
@@ -0,0 +1,114 @@
+package com.pingcap.tispark.overflow
+
+import com.pingcap.tispark.datasource.BaseDataSourceTest
+import org.apache.spark.sql.Row
+import org.apache.spark.sql.types.{StructField, _}
+
+/**
+ * DECIMAL type include:
+ * 1. DECIMAL
```

Comment:
If this suite only contains `Decimal`, I think it is already self-explained.

Reply:
The mapping rule from `Mysql Data Type` to `com.pingcap.tikv.types.DataType` is not so obvious.  So each test in `core/src/test/scala/com/pingcap/tispark/overflow` has such comment  to remind me which `Mysql Data Type` is tested in this test.

Reply:
Then we can rephrase the comment.